Core/include/Acts/Propagator/VoidNavigator.hpp

```diff
@@ -56,6 +58,14 @@ class VoidNavigator {
     return nullptr;
   }
 
+  const TrackingVolume* currentVolume(const State& /*state*/) const {
+    return nullptr;
+  }
+
+  const IVolumeMaterial* currentVolumeMaterial(const State& /*state*/) const {
+    return nullptr;
+  }
```

Comment:
My impression was that the steppers stop using these directly. I guess they're still used in the Propagator?

Reply:
right I now depend on this in the propagator